models/createUser.py

```diff
@@ -0,0 +1,68 @@
+from models.database_model import Tutor,Topic, Registered_User
+from sqlalchemy.orm import Session
+from pydantic import BaseModel
+from typing import List
+
+class UserCreate(BaseModel):
+    id: str
+    first_name: str
+    last_name: str
+    email: str
+    password: str
+
+
+class TutorCreate(BaseModel):
+    id: str
+    topics: List[str]
+    cv_link: str
+    description: str
+    classes: str
+    price: float
+    average_ratings: float
+    times_available: str
+    main_languages: str
+    prefer_in_person: bool
+    other_languages: str
+
+def createUser(user: UserCreate, db: Session):
+    new_user = Registered_User(
+        id=user.id,
+        first_name=user.first_name,
+        last_name=user.last_name,
+        email=user.email,
+        password=user.password,
+        admin_status=False,
+        profile_picture_link="fakelink",
```

Comment:
Can you just remove profile_picture_link entirely it's not needed for a basic user.